xscertparser/cmd/acklogparser.py

```diff
@@ -268,12 +268,22 @@ def post_json_to_mongodb(json):
 
 
 def validate_test_run(json):
+    xs_version = json['global_config']['xs_version'].split('.')
+    xs_version = tuple([int(i) for i in xs_version])
+
+    # CP-30556: Deprecate Marvell (Qlogic) legacy drivers for CH 8.0
+    driver_black_list = ["qla4xxx", "qla3xxx", "netxen_nic", "qlge", "qlcnic"]
```

Comment:
I think it would be better to have this list in a separate file ("blacklist.conf" or something similar) which we read in here. Much more obvious then.

Reply:
Understood 100%.
I moved the list to front of py to let it obvious, rather than put into blacklist.conf, because we don't recommend HCL team use one's private configuration, but should just install/use latest acklogparser version with identical listing rules.